.github/actions/deploy/deploy.py

```
@@ -0,0 +1,851 @@
+                returncode=return_code,
+                stdout='\n'.join(output_lines),
+                stderr=''  # Already merged into stdout
+            )
+
+            if check and return_code != 0:
+                raise subprocess.CalledProcessError(
+                    return_code, cmd, output=result.stdout)
+
+            return result
+
+        except subprocess.TimeoutExpired as e:
```

Ruff (0.14.10)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)